### PR TITLE
Feature/color typography styleguide

### PR DESCRIPTION
**Pull request by alex-windett**

### CODE REVIEW

**Comment by alex-windett** [comment]
Seeing if this is worth contining with - just an initial idea that needs refinement https://github.com/yldio/joyent-portal/pull/263/commits/87bdc5345d61bcc1f40a062ad1535a98808db924

**Comment by alex-windett** [comment]
Being thinking of the most logical and readable way to organised colors - i've come up with this object structure. Thoughts??

```
const primary = {
  primary: '#3B47CC',
  primaryHover: '#1838C0',
  primaryActive: '#12279F',
  primaryDestaurated: '#3B4AAF',
  primaryDesaturatedHover: '#34429D',
  primaryDestauratedActive: '#2D3884',
  primaryDark: '#2D3884',
  primaryDarkHover: '#34429D',
  primaryDarkActive: '#2D3884',
};

const secondary = {
  secondary: '#464646',
  secondaryHover: '#3F3F3F',
  secondaryActive: '#343434',
};

const white = {
  white: '#FFFFFF',
  whiteHover: '#F8F8F8',
  whiteActive: '#E9E9E9',
};

const grey = {
  grey: '#D8D8D8',
}

base = {
  ...primary,
  ...secondary,
  ...white,
  ...grey,
  disabled: "#FAFAFA",
  background: "#FAFAFA",
  green: '#00AF66',
  greenDark: '#009858',
  orange: '#E38200',
  orangeDark: '#CB7400',
  red: '#DA4B42',
  redDark: '#CD251B',
};
```

File: ui/src/shared/constants/colors.js [hunk 1]
```diff
@@ -5,7 +5,7 @@
 * there. Lets try and keep different color variations down ot a minimum.
 * */
 
-const base = {
+let base = {
```
Comment (ramitos) [inline ui/src/shared/constants/colors.js]: Why don't you move the `base` declaration some lines below, so that it doesn't need to be `let`?

**Comment by alex-windett** [comment]
I have, above was more just pseudo code

**Comment by alex-windett** [comment]
@ramitos @juditgreskovits Just finalised the typography/composers/colors. Could you guys take a look

**Comment by alex-windett** [comment]
I've used trailing commas as it reduces the lines that are changed in git

**Pull request merged**